main.tf

```diff
@@ -29,7 +32,7 @@ resource "azurerm_subnet" "subnet" {
   name                 = "${var.prefix}TFSubnet"
   resource_group_name  = azurerm_resource_group.rg.name
   virtual_network_name = azurerm_virtual_network.vnet.name
-  address_prefixes     = ["10.0.1.0/24"]
+  address_prefix       = "10.0.1.0/24"
```

Comment:
Is there a reason why you changed it to `address_prefix`? This argument is being deprecated for `address_prefixes`

https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/subnet#argument-reference

![image](https://user-images.githubusercontent.com/7556325/92273394-23bd2d80-eea0-11ea-9d2a-42b122d6f997.png)

Reply:
I see that now, I'll revert